aleph-client/src/connections.rs

```diff
@@ -249,16 +252,18 @@ impl<C: AsConnection + Sync> ConnectionApi for C {
         Iterable: Sync,
     >(
         &self,
-        addrs: &StaticStorageAddress<T, Yes, Defaultable, Iterable>,
+        addrs: &Address<StaticStorageMapKey, T, Yes, Defaultable, Iterable>,
         at: Option<BlockHash>,
-    ) -> Option<T::Target> {
+    ) -> Option<T> {
         info!(target: "aleph-client", "accessing storage at {}::{} at block {:?}", addrs.pallet_name(), addrs.entry_name(), at);
-        self.as_connection()
-            .as_client()
-            .storage()
-            .fetch(addrs, at)
-            .await
-            .expect("Should access storage")
+
+        let storage = self.as_connection().as_client().storage();
```

Comment:
Api change in subxt in 0.28 - storage.at(Option<block_hash>) has changed to `.at(block_hash)` and `.at_latest()`. Should we also change our api.